doc/specs/vulkan/appendices/VK_NVX_device_generated_commands.txt

```diff
@@ -174,7 +177,7 @@ Provide full flexibilty for code:indirectCommands.
 5) Should the input be provided as SoA or AoS?
 
 It is desired by application to reuse the list of objects and render them
-with some kind override.
+with some kind of a override.
```

Comment:
nit: "of *an* override"

Reply:
good eye

Reply:
"desired by application" is weird too.
I am not quite sure what it talks about, but seems like it should be "desirable for the application".